android/library/src/main/java/io/logto/android/Logto.kt

```diff
@@ -1,19 +1,82 @@
 package io.logto.android
 
+import android.app.Application
 import android.content.Context
 import io.logto.android.authflow.webview.WebViewAuthFlow
+import io.logto.android.callback.AuthenticationCallback
+import io.logto.android.config.LogtoConfig
 import io.logto.android.model.Credential
 import io.logto.android.storage.CredentialStorage
 
-class Logto {
-    companion object {
-        private var credentialStorage: CredentialStorage? = null
+class Logto private constructor() {
+
+    private lateinit var logtoConfig: LogtoConfig
+
+    private lateinit var application: Application
+
+    private var inited: Boolean = false
+
+    private var useStorage: Boolean = false
+
+    private var credentialStorage: CredentialStorage? = null
+
+    private var _credential: Credential? = null
+
+    val credential: Credential?
+        get() = credentialStorage?.getCredential() ?: _credential
+
+    fun init(
+        application: Application,
+        logtoConfig: LogtoConfig,
+        useStorage: Boolean = false
```

Comment:
should the default be true? or we can just use `CredentialStorage` unless the user will meet specific block

Reply:
we will introduce more options in the future: [LOG-111](https://linear.app/silverhand/issue/LOG-111/custom-options-for-logto-initialize)